src/screens/login/LoginScreen.tsx

```diff
@@ -0,0 +1,144 @@
+        <View style={LoginScreenStyles.logoBody}></View>
+        <Text style={LoginScreenStyles.h1}>
+          Hey there<Text style={LoginScreenStyles.blueText}>.</Text>
+        </Text>
+        <Text style={LoginScreenStyles.h2}>We're glad you're back.</Text>
+        <View style={LoginScreenStyles.emailContainer}>
+          <Controller
+            control={control}
+            render={({ field: { onChange, onBlur, value } }) => (
+              <TextInput
+                style={LoginScreenStyles.emailInput}
+                onBlur={onBlur}
+                onChangeText={value => onChange(value)}
+                value={value}
+                placeholder='Email'
+              />
+            )}
+            name='email'
+            rules={{ required: true }}
+          />
+          <View style={LoginScreenStyles.passwordContainer}>
+            <Controller
+              control={control}
+              render={({ field: { onChange, onBlur, value } }) => (
+                <TextInput
+                  style={LoginScreenStyles.passwordInput}
+                  onBlur={onBlur}
+                  onChangeText={value => onChange(value)}
+                  value={value}
+                  placeholder='Password'
+                  secureTextEntry={isPasswordHidden}
+                />
+              )}
+              name='password'
+              rules={{ required: true }}
+            />
+            <Pressable
+              style={LoginScreenStyles.viewPasswordButton}
+              onPress={() => setIsPasswordHidden(!isPasswordHidden)}>
+              {isPasswordHidden ? (
+                <Entypo
+                  name='eye-with-line'
+                  size={24}
+                  color={COLORS.NIGHTLIGHT_GRAY}
+                />
+              ) : (
+                <Entypo name='eye' size={24} color={COLORS.NIGHTLIGHT_GRAY} />
+              )}
+            </Pressable>
+          </View>
+
+          <View style={LoginScreenStyles.forgotPasswordContainer}>
+            <Text style={LoginScreenStyles.forgotPassword}>
+              Forgot password?
+            </Text>
+          </View>
+          <Pressable
+            style={LoginScreenStyles.signInButton}
+            onPress={handleSubmit(onSubmit)}>
+            <Text style={LoginScreenStyles.signInButtonText}>Sign In</Text>
+          </Pressable>
+          <Text style={LoginScreenStyles.continueWithText}>
+            Or continue with
+          </Text>
+          <Pressable style={LoginScreenStyles.googleSignInButton}>
+            <Image
+              source={require('../../../assets/googleIcon.png')}
+              style={LoginScreenStyles.googleIcon}
+            />
+            <Text style={LoginScreenStyles.googleSignInButtonText}>
+              Sign in with Google
+            </Text>
+          </Pressable>
+          <View>
+            <Text style={LoginScreenStyles.notMemberText}>
+              Not a member?&nbsp;&nbsp;
```

Comment:
```suggestion
              Not a member?
```
idk what those chars are